css/selectors/-moz-submit-invalid.json

```diff
@@ -15,12 +15,40 @@
             "edge": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": "4"
-            },
-            "firefox_android": {
-              "version_added": "4"
-            },
+            "firefox": [
+              {
+                "version_added": "88",
+                "notes": "From Firefox 88 the feature has been placed behind a flag. See <a href='https://bugzil.la/1694129'>bug 1694129</a>.",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "layout.css.moz-submit-invalid.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "4",
+                "version_removed": "88"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "88",
```

Comment:
Oh, I'm sorry, @rachelandrew but I just remembered something significant: I learned last week that Firefox for Android doesn't provide access to preferences in regular releases (see https://github.com/mdn/browser-compat-data/issues/9679). Can we remove this support statement?

If you still think we ought to provide the bug link, we could add a note to the other support statement saying something like, `From version 88, the feature has been withdrawn. See <a href='https://bugzil.la/1694129'>bug 1694129</a>.`

Reply:
@rachelandrew hi! Can you finalize this one?